RxGesture/RxGesture.xcodeproj/project.pbxproj

```diff
@@ -493,7 +493,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				FRAMEWORK_SEARCH_PATHS = "../Carthage/Build/**";
+				FRAMEWORK_SEARCH_PATHS = ../Carthage/Build/iOS;
```

Comment:
No, it does not break macOS target.
As for me, on 2 different macs running Sierra and High Sierra I can't build Carthage without this change.